### ISSUE TITLE
Suggestion: Mantain a conversation with Monika!

### ISSUE THREAD

**Issue by Reclaimer1905** [Backdash/MonikaModDev#630]
I was Just spending time with Monika (That sounds a little bit strange now haha) and i realized that there are a ton of things she ask to us and we can answer to just some of them. And i was thinking about implementing a conversation system (something like a chat bot, just similar not exactly a chat bot) so we can actually talk with Monika.

I don't actually know if it's too hard to get on MAS, or if Ren'Py can't stand it; but if it's possible, maybe we should consider it for future builds (?)

Note: Sorry if i didn't said anything well or something like that, my english isn't perfect yet. But if someone have any question about this, feel free to ask!

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
This have already been brought up so many times I won't even bother referencing those issues.
tl;dr: making a life-like chatbot able to believably simulate a personality (much less a personality of somebody as well educated and reasonable as Monika) is not a trivial task not only for a fan project maintained by ~50 people but for a modern science as a whole.

**Comment by seriousCoolness** [comment]
I don't think it necessarily has to simulate a personality. I think that if there were certain dialogs that had chances for continuity, it'd be enough. 
For example, Monika often asks you things at the end of her topics, so maybe a system could be implemented to interpret whether you say something related to that question, and if so, Monika would talk a bit more about the topic briefly, (using a pre-written line, of course,) and at the end of her dialog, this time she doesn't ask anything, not even rhetorically, prompting the player that the conversation is over.
I think it could work similar to how she selects the topics now, actually, given that most of her questions reference the topic's key-words in some form.

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
That would be somewhat complex (for example if you allow the player to agree or disagree with what she says in her topic you will have to write her responses for both cases and then you suddenly have thrice the amount of script for each topic) and is no more relevant since the talk system is being replaced.

**Comment by Winblows2** [comment]
Closing, but we are working on replacing the current chatbot with a prompt system.
